### PR TITLE
PMM-6476 mongodb ssl

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Update of MongoDB SSL info with available params taken from 'pmm-admin add mongodb --help'

### CODE REVIEW

File: docs/manage/client-mongodb-ssl.md [hunk 1]
```diff
@@ -9,32 +9,20 @@ the MongoDB monitoring service.
...
+`--tls`
+: Enable a TLS connection with mongo server
 
-`--mongodb.tls-ca`  (string)
-:   A path to a PEM file that contains the CAs that are trusted for server connections.
+`--server-insecure-tls`
+: Skip PMM Server TLS certificate validation
 
-    *If provided*: MongoDB servers connecting to should present a certificate signed by one of these CAs.
+`--tls-skip-verify`
```
Comment (BupycHuk) [inline docs/manage/client-mongodb-ssl.md:23]: I'm not sure that it should be here

File: source/manage/client-mongodb-ssl.rst [hunk 2]
```diff
@@ -16,32 +16,19 @@ Run this command as root or by using the ``sudo`` command
...
-   A path to a PEM file that contains the CAs that are trusted for server connections.
-
-   *If provided*: MongoDB servers connecting to should present a certificate signed by one of these CAs.
-
-   *If not provided*: System default CAs are used.
-
-``--mongodb.tls-cert`` (string)
-   A path to a PEM file that contains the certificate and, optionally, the private key in the PEM format. This should include the whole certificate chain.
+``--server-insecure-tls``
+   Skip PMM Server TLS certificate validation
```
Comment (BupycHuk) [inline source/manage/client-mongodb-ssl.rst:27]: And it too...

File: docs/manage/client-mongodb-ssl.md (hunk 1 above)
Comment (PaulJacobs-percona) [reply docs/manage/client-mongodb-ssl.md:23]: Are you saying remove the tls-skip-verify option?

Comment (BupycHuk) [reply docs/manage/client-mongodb-ssl.md:20]: yes

Comment (BupycHuk) [reply docs/manage/client-mongodb-ssl.md:20]: no, it was about the flag above. looks like I placed comment in a wrong place.

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**